src/templates/store-settings/paymentcurrencies/index.html

```diff
@@ -21,7 +21,7 @@
 {% for currency in currencies %}
 	{% set tableData = tableData|merge([{
         id: currency.id,
-        title: currency.primary ? "{name} (Primary)"|t('commerce', { name: currency.name }) : currency.name,
+        title: currency.primary ? "{name} (Primary)"|t('commerce', { name: currency.name })|e : currency.name|t('site'),
```

Comment:
`|e` for `currency.name` if it's user definable?

Reply:
Auto escaped in the title column